CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ include("cmake/pgm_version.cmake")
 
 project (power_grid_model VERSION ${PGM_VERSION})
 
-option(PGM_ENABLE_TESTS "Enable tests" OFF)
+option(PGM_ENABLE_DEV_BUILD "Enable developer build (e.g.: tests)" OFF)
```

Comment:
this is a bugfix

note to the reviewer: please double-check that this is indeed the option added to the `CMakePresets.json` and that it is the one used to enable tests later in this file

Reply:
This option is not used at all in any places. So just remove this line.

Reply:
It is used by CMakePresets.json to enable dev build. Without this, only the C API is built by default, not the tests. With it enabled, the tests are also built. This was introduced to allow a better default experience for people who want to build from source with a minimal set of features enabled.

This bugfix fixes the mismatch between the CMake option and the actual CMake variable used in CMakeLists.txt to enable dev build

Reply:
I mean, this particular `option` line is not used. Without this, it still works as intended. When user or preset defines `PGM_ENABLE_DEV_BUILD`, developer build is enabled, otherwise only the C-API.

Reply:
still, it is probably a good idea to tell users that and how it is possible to enable the DEV build without digging through the code base. `option` is designed specifically for that. From a best-practice point of view, I do think it's a good idea to keep it. What is your reasoning for explicitly removing it?

Reply:
You are correct. I did not know the meaning of `option` in `cmake`.